src/prefect/task_worker.py

```diff
@@ -411,7 +440,7 @@ async def serve(
     Example:
         ```python
         from prefect import task
-        from prefect.task_worker import serve
+        from prefect.task_worker import aserve
```

Comment:
It doesn't look like an `aserve` was introduced in this PR?
```suggestion
        from prefect.task_worker import serve
```